### DISCUSSION TITLE
Sending Emails to the platform users

### DISCUSSION THREAD

**Discussion by x-R4Z0R-x** [dfir-iris/iris-web#436] [Ideas]
Hi,

Can a user get an email whenever a task/case is assigned to him?
Also for status updates
If yes, how can I configure this?
I looked for anything related to this but with no luck

Thanks in advance

**Comment by x-R4Z0R-x** [comment]
Hi @whikernel ,
Can you please check this? is it possible?

**Reply by skeet16** [reply]
Any updates on this?

**Reply by x-R4Z0R-x** [reply]
@skeet16 
Not yet, the dev team didn't respond

**Comment by whikernel** [comment]
Hi @x-R4Z0R-x  

Apologize for the delay - we mainly use discord for the discussions and I kinda miss quite a lot here.  
There is currently no native way to notify a user by email. 
However you can use IRIS Webhook module and an automation tool / script to achieve the same result.

**Comment by x-R4Z0R-x** [comment]
Thanks
